trace.go

```diff
@@ -32,6 +32,8 @@ import (
 
 const errParameter = tool.Error("parameter are nil")
 
+var emptyTraceID = []int64{0, 0, 0}
```

Comment:
How about declaring it as constant

Reply:
And make it accessible from another package

Reply:
> How about declaring it as constant

Constants cannot be array.

Ref: 
1. https://stackoverflow.com/questions/13137463/declare-a-constant-array
1. https://blog.golang.org/constants

Reply:
I would like to modify it to keep it consistent with the Java agent.
1. activeSpan == nil -> N/A
1. activeSpan == NoopSpan -> [Ignored Trace]
1. activeSpan == segmentSpan -> part1.part2.part3

Reply:
Does go2sky could work in no backend service? I don't remember this part, could you recheck?

Reply:
> Does go2sky could work in no backend service? 

`Reporter` should work when not registered,`TestTrace_TraceID ` covers all three

Reply:
OK. If that is supported, it makes sense. But then, trace Id should be a string.

Reply:
Done.